whois-update/src/main/java/net/ripe/db/whois/update/dns/zonemaster/ZonemasterDnsGateway.java

```diff
@@ -37,7 +37,7 @@ public class ZonemasterDnsGateway implements DnsGateway {
     private static final Logger LOGGER = LoggerFactory.getLogger(ZonemasterDnsGateway.class);
 
     private static final int TEST_PROGRESS_SLEEP_SECONDS = 5;
-    private static final int TEST_PROGRESS_MAXIMUM_RETRIES = ((60 * 5) / TEST_PROGRESS_SLEEP_SECONDS);
+    private static final int TEST_PROGRESS_MAXIMUM_RETRIES = ((60 * 10) / TEST_PROGRESS_SLEEP_SECONDS);
```

Comment:
We have timeout of 5 mins for requests from db-web-ui. Will this help if we wait 10 mins? User still gets 500 Internal server error

Reply:
We can also extend the read timeout in db-web-ui for the domain object wizard *only* to 10 minutes. I tested this works OK.